fixtures/nicols/beam-led-10.json

```
@@ -0,0 +1,84 @@
+{
+  "$schema": "https://raw.githubusercontent.com/OpenLightingProject/open-fixture-library/master/schemas/fixture.json",
+  "name": "Beam Led 10",
+  "shortName": "LED",
+  "categories": ["Fan", "Color Changer", "Strobe"],
```

```suggestion
  "categories": ["Color Changer", "Effect"],
```

I'm not entirely sure what this is, but it doesn't seem to be a Fan.